docs/deployment_patterns/how_to_use_gx_with_aws/how_to_use_gx_with_aws_using_cloud_storage_and_pandas.md

```diff
@@ -0,0 +1,325 @@
+
+<!-- 1.4.4 (Optional) Copy existing Expectation JSON files to the Amazon S3 bucket -->
+
+import OptionalCopyExistingExpectationsToS3 from '@site/docs/guides/setup/configuring_metadata_stores/components_how_to_configure_an_expectation_store_in_amazon_s3/_copy_existing_expectation_json_files_to_the_s_bucket_this_step_is_optional.mdx'
+
+<!-- 1.4.5 (Optional) Verify that copied Expectations can be accessed from Amazon S3 -->
+
+import OptionalVerifyCopiedExpectationsAreAccessible from '@site/docs/guides/setup/configuring_metadata_stores/components_how_to_configure_an_expectation_store_in_amazon_s3/_confirm_that_expectations_can_be_accessed_from_amazon_s_by_running_great_expectations_suite_list.mdx'
+
+<!-- 1.5 Configure your Validation Results Store on Amazon S3 -->
+
+<!-- 1.5.1 Identify your Data Context's Validation Results Store -->
+
+import IdentifyDataContextValidationResultsStore from '@site/docs/guides/setup/configuring_metadata_stores/components_how_to_configure_a_validation_result_store_in_amazon_s3/_identify_your_data_context_validation_results_store.mdx'
+
+<!-- 1.5.2 Update your configuration file to include a new Store for Validation Results on Amazon S3 -->
+
+import AddS3ValidationResultsStoreConfiguration from '@site/docs/guides/setup/configuring_metadata_stores/components_how_to_configure_a_validation_result_store_in_amazon_s3/_update_your_configuration_file_to_include_a_new_store_for_validation_results_on_s.mdx'
+
+<!-- 1.5.3 Verify that the new Amazon S3 Validation Results Store has been added successfully -->
+
+import VerifyS3ValidationResultsStoreExists from '@site/docs/guides/setup/configuring_metadata_stores/components_how_to_configure_a_validation_result_store_in_amazon_s3/_confirm_that_the_new_validation_results_store_has_been_added_by_running_great_expectations_store_list.mdx'
+
+<!-- 1.5.4 (Optional) Copy existing Validation results to the Amazon S3 bucket -->
+
+import OptionalCopyExistingValidationResultsToS3 from '@site/docs/guides/setup/configuring_metadata_stores/components_how_to_configure_a_validation_result_store_in_amazon_s3/_copy_existing_validation_results_to_the_s_bucket_this_step_is_optional.mdx'
+
+<!-- 1.6 Configure Data Docs for hosting and sharing from Amazon S3 -->
+
+<!-- 1.6.1 Create an Amazon S3 bucket for your Data Docs -->
+import CreateAnS3BucketForDataDocs from '@site/docs/guides/setup/configuring_data_docs/components_how_to_host_and_share_data_docs_on_amazon_s3/_create_an_s3_bucket.mdx'
+
+<!-- 1.6.2 Configure your bucket policy to enable appropriate access -->
+import ConfigureYourBucketPolicyToEnableAppropriateAccess from '@site/docs/guides/setup/configuring_data_docs/components_how_to_host_and_share_data_docs_on_amazon_s3/_configure_your_bucket_policy_to_enable_appropriate_access.mdx'
+
```

Comment:
This portion was new to me, and I'm not sure if I was able to configure it correctly. Is there a possibility that additional resources could be included in the doc? (other than the existing link to AWS)